examples/apigateway/main.go

```diff
@@ -0,0 +1,203 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	stdlog "log"
+	"net/http"
+	"net/url"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/go-kit/kit/endpoint"
+	addsvc "github.com/go-kit/kit/examples/addsvc/client/grpc"
+	"github.com/go-kit/kit/examples/addsvc/server"
+	"github.com/go-kit/kit/loadbalancer"
+	"github.com/go-kit/kit/loadbalancer/consul"
+	log "github.com/go-kit/kit/log"
+	//grpctransport "github.com/go-kit/kit/transport/grpc"
+	httptransport "github.com/go-kit/kit/transport/http"
+	//proto "github.com/golang/protobuf/proto"
+	"github.com/gorilla/mux"
+	"github.com/hashicorp/consul/api"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	fs := flag.NewFlagSet("", flag.ExitOnError)
```

Comment:
No reason to use a flagset, as we're not importing any package with a transitive dependency on glog. You can just use flag.String, flag.Parse, etc.